fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/ContainerServiceImpl.java

```diff
@@ -61,10 +63,22 @@ public Container findOrCreate(final Session session, final String path) {
 
             if (node.isNew()) {
                 initializeNewObjectProperties(node);
+
+                final Node parent = node.getDepth() > 0 ? node.getParent() : null;
+
+                if (parent != null) {
+                    touch(parent);
+                    touchLdpMembershipResource(node);
+                }
+            }
+
+            if (node.isNew()) {
+                touch(node);
             }
 
             return new ContainerImpl(node);
         } catch (final RepositoryException e) {
+            LOGGER.info("Error: {}", e);
```

Comment:
The general principle here is that stacktraces are not useful for the vast majority of users, and they dramatically increase logfile sizes.

I agree with @ajs6f that I like to see the stacktraces, and find them useful for debugging.  This issue comes up fairly frequently in code review.  Maybe we should have a debug mode that enables dumping stacktraces?